ios/MaasNearbyTransit/README.md

```diff
@@ -0,0 +1,3 @@
+# MaasNearbyTransit
+
+A description of this package.
```

Comment:
🙈 I'd rather we didn't have READMEs at all if they're not saying anything useful.

Reply:
It feels like the description could contain a snapshots and some explanation where elements are used. What do you think ? 🤔

Reply:
Yeah, nice idea to include the snapshots! Easy and impactful.

Reply:
Check it out.